addons/mrp/tests/test_manual_consumption.py

```diff
@@ -230,3 +230,21 @@ def get_moves(mo):
         move_auto, move_manual = get_moves(backorder)
         self.assertEqual(move_auto.manual_consumption, False)
         self.assertEqual(move_manual.manual_consumption, True)
+
+    def test_update_manual_consumption(self):
+        """
+        Check that the manual consumption is set to true when the quantity is manualy set.
+        """
+        mo_form = Form(self.env['mrp.production'])
+        mo_form.product_id = self.bom_3.product_id
+        mo_form.bom_id = self.bom_1
+        mo_form.product_qty = 1
+        mo = mo_form.save()
+        mo.action_confirm()
+        with Form(mo) as fmo:
+            with fmo.move_raw_ids.edit(0) as line_0:
+                line_0.quantity = 3.0
+                line_0.picked = True
+        mo = fmo.save()
```

Comment:
```suggestion
```
This will be already one when exiting the `with`
https://github.com/odoo/odoo/blob/4251995c04056a71c60cd39dbd5aff22d308ecd1/odoo/tests/form.py#L400-L402